CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 2.4.0 (unreleased)
 ------------------
 
+2203
+- #2203 Added missing parameter date to date_to_string on sample listing
```

Comment:
Can you remove that orphan `2203` from top and replace the description to something more descriptive from the end-user point of view, e.g. *Fix date sampled is empty in listings when sampling workflow is enabled* ?

Reply:
Thanks @xispa, put on required changes.